lib/ecto_materialized_path/path.ex

```diff
@@ -7,7 +7,7 @@ defmodule EctoMaterializedPath.Path do
   """
 
   def cast(list) when is_list(list) do
-    path_is_correct? = Enum.all?(list, fn(path_id) -> is_integer(path_id) end)
+    path_is_correct? = Enum.all?(list, fn(path_id) -> (is_integer(path_id) || is_binary(path_id)) end)
```

Comment:
Please  write an option to `use` macro, something like

```
use EctoMaterializedPath,
  id_type: :uuid # default is `:regular`
```

and add a few lines to the doc

Reply:
thx for the feedback, I like the way you want it to do. what I'm struggling (Elixir noob) is how to access the `id_type` passed to `use` from within the Path module. 
Thought about adding a function to the base module, but then I don't know how to access the `id_type` outside of the macro.

Reply:
You have to figure out by yourself. I don't have a time right now :( Use SO to get it and write according to my feedback

Reply:
ok.